### PR TITLE
Added github actions to build Freemarker on ubuntu 18.04

### PR DESCRIPTION
**Pull request by HanSolo**
More or less a test to run the ant build on each push using github actions.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on: [push]
```
Comment (vdiravka) [inline .github/workflows/ci.yaml:3]:
```suggestion
on: [push, pull_request]
```

Comment (vdiravka) [reply .github/workflows/ci.yaml:3]: consider adding:
`timeout-minutes: 5`

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
```
Comment (vdiravka) [inline .github/workflows/ci.yaml:12]:
```suggestion
        uses: actions/setup-java@v2
```

**Review by vdiravka — COMMENTED** [review]
+1 (not binding)

**Comment by ddekany** [comment]
There's now another CI Github Action merged, so closing this one.

**Pull request closed without merging**